automation_testing/tests/Components/dataview/advanced_filters.spec.ts

```diff
@@ -430,4 +430,76 @@ test.describe.parallel("Components - Data View - Advanced Filters", () => {
 		await advancedFilters.waitForElementLoad();
 		expect(await pagination.paginationValue.textContent()).toBe(`1-${dataview_data.resultPerPageDefault} of ${dataview_data.totalRecords}`);
 	});
+
+	test("Validate the Created filter styles.", async () => {
```

Comment:
After seeing the difference between these 2 tests, they seem to be exactly the same, the only thing that changes is the "advancedFilters.createdOption" and "advancedFilters.updatedOption". Let's put all in a function that receives the type. For example: async  testOptions(option) {  ... await advancedFilters[option].check(); ...  }